packages/website/docs/networks-and-flows/ChordDiagram.mdx

```diff
@@ -0,0 +1,124 @@
+    target: number | string | ChordInputNode;
+}
+
+type GraphData<ChordInputNode, ChordInputLink> = {
+  nodes: ChordInputNode[];
+  links: ChordInputLink[];
+}
+```
+
+## Node Customization
+### Node Label
+You can supply node labels through that data as a property of `ChordInputNode`, or by using a `StringAccessor` function in the component:
+<DocWrapper {...defaultProps()} nodeLabel={d => `${d.id}`}/>
+
+### Node Label Alignment
+By default, the labels will be placed inside the node with respect to the curve. Alternatively, you can set `nodeLabelAlignment` to _perpendicular_:
+<DocWrapper {...defaultProps()} nodeLabel={d => `${d.id}`} nodeLabelAlignment="perpendicular"/>
+
+### Node Color
+<DocWrapper {...defaultProps()} nodeColor={d => d.group === 'groupX' ? '#1af' : '#a23' }/>
+
+## Link Customization
+### Curve Types
+You can customize the shape of your _Link_ to the `curveType` property, which accepts a `string` or `CurveType` value:
+<InputWrapper {...defaultProps(5)}
+inputType="select"
+property="curveType"
+options={["linearClosed", "step", "catmullRom"]}
+defaultValue="step"/>
+
+## Further Customization
+Like the [_Donut_](/docs/misc/donut) component, you can change the appearance of your circular chart with the following related properties:
+
+- [`angleRange`](/docs/misc/Donut/#angle-range)
+- [`cornerRadius`](/docs/misc/Donut/#corner-radius)
+- [`padAngle`](/docs/misc/Donut/#pad-angle)
+
+## Hierarchies
+The _ChordDiagram_ supports hierarchal data. You can provide set `nodeLevels` to the list of
+properities you want to group your data by. Consider data with the following properties:
+```ts
+const data = {
+  nodes: [
+    { id: 'A', level: 'moderate', group: 'groupX', ... },
+    { id: 'B', level: 'high', group: 'groupY', ...},
+    ...
+  ],
+  links: [...]
+}
+```
+Then, by providing this list we can see three distinct layers in the visualization:
+<DocWrapper {...defaultProps(20)} height={450} nodeLevels={['level', 'group']}/>
+
+## Radius Scale Exponent
+The `radiusScaleExponent` property affects how the radii scale with each hierarchy level. Increasing this value will result in more distance
+between each level, while decreasing results in the opposite. Default = `2`.
+<InputWrapper {...defaultProps(15)} height={450} nodeLevels={['level', 'group']} inputType="range" defaultValue={0.75} property="radiusScaleExponent" inputProps={{ min: 0.25, max: 3, step: 0.25 }}/>
+
+## Events
+```ts
+import { ChordDiagram, ChordNode, ChordRibbon } from '@volterra/vis'
+
+events = {
+  [ChordDiagram.selectors.node]: {
+    mouseover: (d: ChordNode<N>) => {},
+    mouseout: (d: ChordNode<N>) => {},
+  },
+  [ChordDiagram.selectors.link]: {
+    mouseover: (d: ChornRibbon<N>) => {},
+    mouseout: (d: ChornRibbon<N>) => {},
+  },
+}
+```
+<DocWrapper {...defaultProps()} events={{}} excludeGraph showContext={undefined}/>
+
+## CSS Variables
+<details open>
+  <summary>All supported CSS variables</summary>
```

Comment:
I've started providing default values here as well, I think it's useful to show them because there's no easy way to find them. You can use other docs pages as an example.